plans/dockerregistry.tf

```diff
@@ -0,0 +1,119 @@
+resource "azurerm_resource_group" "dockerregistry" {
+  name     = "${var.prefix}dockerregistry"
+  location = "${var.location}"
+  tags {
+    env = "${var.prefix}"
+  }
+}
+
+resource "azurerm_storage_account" "dockerregistry" {
+    name                = "${var.prefix}dockerregistry"
+    resource_group_name = "${azurerm_resource_group.dockerregistry.name}"
+    location            = "${var.location}"
+    account_type        = "Standard_GRS"
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_template_deployment" "dockerregistry"{
+  name  = "${var.prefix}dockerregistry"
+  resource_group_name = "${ azurerm_resource_group.dockerregistry.name }"
+  parameters = {
+	registryName = "${var.prefix}registry"
+ 	registryLocation = "${var.location}"
+	registryApiVersion = "2016-06-27-preview"
+	storageAccountName = "${ azurerm_storage_account.dockerregistry.name }"
+	storageAccountType = "${ azurerm_storage_account.dockerregistry.account_type }"
+	#adminUserEnabled = true	
+  }
+  deployment_mode = "Incremental"
+  template_body = <<DEPLOY
```

Comment:
I think to make our lives easier, all our ARM templates should be put into a separate directory, perhaps `./arm-templates` in the root directory?